chain-network/proto/watch.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+import "types.proto";
+
+package iohk.chain.watch;
+
+message BlockSubscriptionRequest {}
+
+message TipSubscriptionRequest {}
+
+message MempoolSubscriptionRequest {}
+
+message SyncMultiverseRequest {
+  // Block IDs to start synchronization from.
+  //
+  // The client should include the ID of the latest block at or below the
+  // stability depth, as well as tips of any branches known to it from
+  // previously synchronized node state.
+  //
+  // The server should prune the block IDs in the list, discarding any block IDs
+  // not found in the blockchain storage (which might be on discarded branches),
+  // as well as any blocks that are ancestors of other blocks identified in
+  // the list. The server responds with a stream sending any
+  // valid blocks on the (possibly forked) chain that are descendants of
+  // the remaining blocks in the list, unless the list is empty, in which case
+  // the server sends all blocks starting from genesis.
+  // The stream should uphold the parent-to-child order of the blocks,
+  // i.e. no block gets sent before its parent.
+  repeated bytes from = 1;
+}
+
+service Watch {
```

Comment:
If you don't have a trusted node to connect to and you don't want to start a passive node to join the topology and have information coming in from multiple sources. Btw, the latter is fine and is easier to reason with I think

Reply:
I mean, it could very well be useful for something, I'm not saying it isn't. The tip subscription could be used from multiple nodes to build a pool comparison service or something, not so sure about the blocks.

With the current api you could sync from multiple nodes though, just that not in parallel. If they both have the same branches and you provide them to the other one in the sync request then the answer should be empty. To do it in parallel yeah, you would need a `GetBranches` method.